cirq/testing/consistent_decomposition.py

```diff
@@ -37,4 +39,6 @@ def assert_decompose_is_consistent_with_unitary(val: Any):
     actual = circuits.Circuit.from_ops(dec).to_unitary_matrix(
         qubit_order=qubits)
 
-    assert np.allclose(actual, expected, atol=1e-8)
+    lin_alg_utils.assert_allclose_up_to_global_phase(actual,
```

Comment:
Add an `including_global_phase` boolean parameter that determines whether this or `np.allclose` is used.